<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLayerWindow</class>
 <widget class="QDialog" name="AddLayerWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add layer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="layerTypeTabMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="rasterLayerTab">
    <attribute name="title">
     <string>Raster</string>
    </attribute>
    <widget class="QComboBox" name="mapTypeBox_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>82</y>
       <width>408</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Open Street Maps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mapbox Bright</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mapbox Control Room</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stamen</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="mapTypeLabel_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Map type</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="mapLayerName_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="mapLayerNameLabel_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter layer name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addRasterLayerButton">
     <property name="geometry">
      <rect>
       <x>423</x>
       <y>122</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Layer</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="vectorLayerTab">
    <attribute name="title">
     <string>Vector</string>
    </attribute>
    <widget class="QPlainTextEdit" name="vectorLayerName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="vectorLayerNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter layer name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="vectorFilePathName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>405</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="vectorFileNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter file path (only geojson now)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openVectorFileButton">
     <property name="geometry">
      <rect>
       <x>452</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addVectorLayerButton">
     <property name="geometry">
      <rect>
       <x>423</x>
       <y>155</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Layer</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
